images/vscode-ssh-server/root/etc/ssh/sshd_config.d/overrides.conf

```diff
@@ -0,0 +1,10 @@
+Port 2222
+# Disallow password auth
+PasswordAuthentication no
+# Enables VSCode access to the code-server instance it runs, via the ssh tunnel
+AllowTcpForwarding yes
+# Only allow our intended user
+AllowUsers dev
+# Force sessions to start in the /data directory
+ForceCommand bash -c 'cd /data && if [ -n "$SSH_ORIGINAL_COMMAND" ]; then eval "$SSH_ORIGINAL_COMMAND"; else exec bash; fi'
```

Comment:
If we're both migrating to fish...

Reply:
I thought I'd keep the dependencies semi-slim at least :) Since this is intended just for VSCode editing and not the whole development lifecycle that you see in devcontainers, I didn't expect much terminal usage. Deploying devcontainers as a sidecar would a cool alternative to this though!